SETUP.md

```diff
@@ -388,11 +388,20 @@ Wondering where to start?  See our [contribution guidelines](CONTRIBUTING.md) fo
 ---
 ### Bundle Install Tips
 
-#### Apple Silicon (M1) bundle install steps
+#### Apple Silicon (M1) or Intel Mac bundle install steps
 
-On Apple Silicon, additional steps are required to get `bundle install` to work.
+On Apple Silicon/Intel Mac, additional steps are required to get `bundle install` to work.
 
-In Gemfile.lock, replace the two occurrences of libv8 (8.4.255.0) with libv8-node (15.14.0.0). Also update mini_racer to 0.4.0 (from 0.3.1):
+Simply run (if you're having issues only with part of gems in the command - you can run it with just needed gems)
+```
+bundle update libv8 mini_racer
+```
```

Comment:
I believe the existing `SETUP.md` works on Apple Silicon, but people (including myself) often miss a key step:
`bundle config --local without staging test production levelbuilder`

Its in there, but its easy to miss.

This obviously only lets you build in the development mode, but it does work because the mini_racer gem is defined in the `Gemfile` as:
```
gem 'mini_racer', group: [:staging, :test, :production, :levelbuilder]
```
I would suggest we replace this section with a searchable version of the error message, and a reminder to run `bundle config --local without staging test production levelbuilder` prior to trying bundle install.

Reply:
That said, if we **do** keep the existing section, I think we should just completely **replace** the instructions to edit Gemfile.lock with @levadadenys 's suggested: 
````
bundle update libv8 mini_racer
```

Reply:
Oh, and @levadadenys is 100% right that this is not apple silicon specific, its actually related to newer MacOS versions.

Reply:
I don't think its fair to call out MacOS here. Our macos specific instructions are only 20% longer than ubuntu, and I believe a much larger usebase internally. After removing conservative recommendation but no longer required rosetta ask for apple silicon it'll only be 10% longer than ubuntu.

We only have two gems (not in dev) that give MacOS any trouble at this point, and I believe (did I forget something? I well may have?) we already did the hard lift for updating these. Can't we upgrade to libv8-node at this point now that servers are on node 18.04?

Reply:
Also, after a few `SETUP.md` text updates that I just haven't done, we'll have no divergence between Apple Silicon and x86 instructions, and those are currently the only two flavor variants we have in SETUP.md

Reply:
Node wasn't the blocker to updating those libraries; ubuntu is.

I'm less concerned about the technical length of the instructions than I am about the frequency with which they need to be referenced; we get a high volume of questions coming in via slack about getting macos installs working, and very few about ubuntu. The improvements you mention do seem useful, though; I'm happy to wait and see how they affect the frequency with which folks need to reach out for extra help.

Reply:
Volume wise, I was under the impression we had significantly more macOS users. But maybe that's not true. They have been historically very complicated, both because there were flavors (going away), and there wasn't enough effort to simplify them (tendency to add notes rather than fix instructions)